packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -1013,7 +1013,7 @@ class Auth {
     async loginIDPUser(parameters: LoginIDPUserParams) {
         const codeVerifier = this.get('code_verifier')
         const code = parameters.code
-        const usid = parameters.usid
+        const usid = this.get('usid')
```

Comment:
We need to use the guest usid in order to run `mergeBasket`

Reply:
Can you tell me more about this change here? We are now ignoring a parameter that is being passed in? Why aren't we doing something like the fallback where `const usid = parameters.usid ||  this.get('usid')` like you did for the redirect uri?